lib/fastlane/plugin/wpmreleasetoolkit/models/app_version.rb

```diff
@@ -0,0 +1,30 @@
+# Fastlane::Models::AppVersion represents a version of an app with major, minor, patch, and build number components.
+module Fastlane
+  module Models
+    class AppVersion
+      attr_accessor :major, :minor, :patch, :build_number
+
+      # Initializes a new AppVersion instance.
+      #
+      # @param major [Integer] the major version number.
+      # @param minor [Integer] the minor version number.
+      # @param patch [Integer] the patch version number.
+      # @param build_number [Integer] the build number.
+      #
+      def initialize(major, minor, patch, build_number)
```

Comment:
I wonder if it wouldn't make sense to also allow that initialize to take a simple `String and it would do the `split('.')` for us.

```ruby
# Initializes a new AppVersion instance.
#
# @param params [String|Array<#to_i>|*#to_i] 
#        - If passed a `String`, it will be split using `.` and each 4 components will be `.to_i`
#        - If passed an `Array`, or a list of more than on parameters, each element will be `.to_i`
#        Then the 4 resulting elements will serve as major, minor, patch and build_number values of the AppVersion
def initialize(*params)
  if params.count == 1 && params.first.is_a?(String)
    # If we passed a single parameter of type String, so do the Split ourselves
    params = params.first.split('.') if params.count == 1 ** params.first.is_a?(String)
    # If we passed a single parameter of type array, make it as if we passed the values of the array individually
    params = *params if params.count = 1 && params.first.is_a?(Array)
    # At that point `params` is an array of the 4 components of the version
    @major, @minor, @patch, @build_number = params.map(&:to_i)
  end
end
```

Reply:
Addressed in https://github.com/wordpress-mobile/release-toolkit/commit/0ee53b7253b930d88cc031951a8e9ac463810ef5